Response-Errors.md

```diff
@@ -0,0 +1,123 @@
+| `AUTHORIZATION_ERROR` | `1111` | Unauthorized request. Invalid API user | You will get this error if your api user authentication fails | Accessing resources which requires api user authorization |
+| `AUTHORIZATION_ERROR` | `1112` | Unauthorized request. Invalid User | You will get this error if your user authentication fails | Accessing resources which requires user authorization |
+
+### 2. Blockchain Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |----------- | ----------- | ---------- |
+| `BLOCKCHAIN_ERROR` | `2001` | Insufficient token amount in escrow contract |  You will get this error if vendor's escrow contract does not have enough tokens to fund a shopper | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2002` | Insufficient wei amount in escrow contract | You will get this error if vendor's escrow contract does not have enough wei to fund a shopper | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2111` | Invalid gas limit | You will get this error if signed transactions gas limit is different that the relevant one provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2112` | Invalid receiver | You will get this error if signed transactions `to` addresses is different from the `to` addresses provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2113` | Invalid Gas Price | You will get this error if signed transactions gasPrice is different from the gasLimit provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2114` | Method ID/function signature mismatch | You will get this error if a shopper signs different function than these provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2115` | Function params does not match! | You will get this error if a shopper signs a function with different parameter/s than equivalent one provided with generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2116` | Array of signed transactions must be provided | You are required to provide signed transactions in order to process fiat/relayed payment | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2117` | Invalid number of signed transactions | On payment creation, you pass generic transactions. When the shopper signs his intent, the number of transactions should be equal to the count of generic transactions | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2118` | Invalid signer. The transactions were not signed by the Shopper's private key | You pass signed transactions in order to process payment. Their signer( public address) should be equal to shopper's `walletAddress` | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2119` | Cannot decode generic transaction data! | You will get this error if a shopper somehow replace signed transactions with fake data | Processing payment |
```

Comment:
`You will get this error if a shopper somehow replace signed transactions with fake data`
->
`You will get this error if a provided signed transaction is not valid or the parameter types provided on payment creation are not valid `